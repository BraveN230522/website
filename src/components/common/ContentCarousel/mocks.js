import NbaSchedule from '@public/images/team-carousel.jpg';

export const mockCarousel = [
  {
    blogpostID: 1,
    imageURL: NbaSchedule,
    blogpostTitle: '2020-21 NBA Schedule released',
    blogpostSummary:
      'A record 48 games will air in primetime in Europe, the Middle East and Africa, with 24 games each on Saturday and Sunday.',
  },
];
