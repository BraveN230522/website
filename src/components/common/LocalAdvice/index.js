import React from 'react';
import { adopt } from 'react-adopt';
import ReactPlayer from 'react-player';
import CardHorizontal from '@components/common/CardHorizontal';
import { formatSlugSegments, LINK_ABSOLUTE } from '@utils/routes';
import {
  LocalAdviceSection,
  LocalAdviceMore,
  LocalAdviceContainer,
  LocalAdviceMediaContainer,
  LocalAdviceTitleGuide,
  LocalAdviceMedia,
  BtnVideoPlay,
  LocalAdviceCardContent,
  LocalAdviceMainImage,
} from './styles';
import InterestDataQuery from './queries';
import InterestQuestions from '../InterestQuestion';

const Data = adopt({
  interestData: ({ render, interestId, interestType }) => (
    <InterestDataQuery interestType={interestType} interestId={interestId}>
      {render}
    </InterestDataQuery>
  ),
});

const sliceRange = hasVideo => {
  return hasVideo ? [0, 2] : [1, 3];
};

const RenderMedia = ({ video, adviceImage, mainImage }) => {
  if (video || adviceImage)
    return (
      <LocalAdviceMediaContainer>
        <LocalAdviceMedia
          adviceImage={video?.imageURL || adviceImage}
          modalContent={<ReactPlayer url={video?.link} width="100%" height="100%" playing />}
        >
          <LocalAdviceTitleGuide>{video.title}</LocalAdviceTitleGuide>

          <BtnVideoPlay name="btnPlay" />
        </LocalAdviceMedia>
      </LocalAdviceMediaContainer>
    );
  if (mainImage?.imageURL && mainImage?.link && mainImage?.title)
    return (
      <LocalAdviceMediaContainer>
        <LocalAdviceMainImage
          adviceImage={mainImage?.imageURL}
          href={mainImage?.link}
          target="_blank"
          rel="noopener noreferrer"
        >
          <LocalAdviceTitleGuide>{mainImage?.title}</LocalAdviceTitleGuide>
        </LocalAdviceMainImage>
      </LocalAdviceMediaContainer>
    );
  return null;
};

const LocalAdvice = ({ cardData, local, adviceImage, className, interestId, interestType, cityName, venueName }) => {
  const linkUrl =
    cityName && venueName
      ? formatSlugSegments(LINK_ABSOLUTE, cityName, venueName, 'fan-tips')
      : formatSlugSegments(LINK_ABSOLUTE, 'venue', interestId, 'fan-tips');
  return (
    <Data interestId={interestId} interestType={interestType}>
      {({ interestData }) => {
        const video = interestData?.length && interestData.filter(item => item.objectType === 'VIDEO')[0];
        const mainImage = !video && interestData?.length && interestData[0];
        // const fallbackVideo = 'https://www.youtube.com/watch?v=-M0RiKff3Ds';
        const parsedInterestData = interestData?.length
          ? interestData
              .map(item => {
                return {
                  id: item.id,
                  image: item.imageURL,
                  title: item.title,
                  info: item.blogpostSummary,
                  link: item.link,
                };
              })
              .slice(...sliceRange(video))
          : cardData;

        if (!(mainImage?.imageURL && mainImage?.link && mainImage?.title) || !video) return null;

        return (
          <LocalAdviceSection
            className={className}
            centered
            title={local ? `Travel advice for ${local}` : ''}
            // more={<LocalAdviceMore>More</LocalAdviceMore>}
          >
            <LocalAdviceContainer>
              <RenderMedia video={video} adviceImage={adviceImage} mainImage={mainImage} />

              <LocalAdviceCardContent>
                {!!parsedInterestData.length &&
                  parsedInterestData.map(item => <CardHorizontal key={item.id} cardData={item} />)}
              </LocalAdviceCardContent>
            </LocalAdviceContainer>
            {/* NOTE: [HOTFIX SWIAM-1234: to be re-used - do not delete */}
            {/* <InterestQuestions
              interestId={interestId}
              interestType={interestType}
              cityName={cityName}
              venueName={venueName}
            /> */}
            <LocalAdviceMore href={linkUrl}>More</LocalAdviceMore>
          </LocalAdviceSection>
        );
      }}
    </Data>
  );
};

export default LocalAdvice;
