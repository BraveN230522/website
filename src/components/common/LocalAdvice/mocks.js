import CityBg from '@images/city-bg.png';
import CityMobileBg from '@images/city-background-mobile.png';

export const mockLocal = {
  title: 'New York City Sports Tickets',
  description:
    'New York City is the city that never sleeps and it can lay claim to being one of the best sporting cities in the world. With teams in every major competitive sport - football, baseball, soccer, basketball and ice hockey, you would be crazy to sleep when you are in the Big Apple! More',
  imageURL: CityBg,
  mobileImageURL: CityMobileBg,
  adviceImage: 'https://i.insider.com/5d0150e06fc9200b5a05f042?width=1136&format=jpeg',
  local: 'NYC',
  tips: [
    {
      id: 3,
      icon: 'parking',
      tip: 'Arthur Ashe Stadium parking?',
      views: 222,
      replies: 2,
    },
    {
      id: 1,
      icon: 'parking',
      tip: 'Best bars near Metlife Stadium?',
      views: 123,
      replies: 13,
    },
    {
      id: 2,
      icon: 'parking',
      tip: 'Traffic around Citi Field?',
      views: 3087,
      replies: 1,
    },
  ],
};

export const mockAdvice = [
  {
    id: 1,
    image:
      'https://content.r9cdn.net/rimg/kimg/f1/0b/34800236-58e51e7e.jpg?width=1004&crop=true&caller=HotelDetailsPhoto.jpeg',
    title: 'Restaurants near Madison Square Garden',
    info:
      'For years the area around Manhattan’s premier entertainment venue has been criticised for a lack of decent food options.',
  },
  {
    id: 2,
    image:
      'https://content.r9cdn.net/rimg/kimg/f1/0b/34800236-58e51e7e.jpg?width=1004&crop=true&caller=HotelDetailsPhoto.jpeg',
    title: 'Hotels near Madison Square Garden',
    info:
      'Now, you might have heard that Manhattan isn’t exactly the cheapest place in the world, and a night or two in a hotel could ..',
  },
];
