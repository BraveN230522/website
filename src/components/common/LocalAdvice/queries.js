import { useQuery } from '@apollo/react-hooks';
import { get } from 'lodash';
import console from '@utils/console';
import { GET_INTEREST_DATA } from '../../../graphql/queries/interest-data';

const InterestDataQuery = ({ children, interestId, interestType }) => {
  const { loading, error, data } = useQuery(GET_INTEREST_DATA, {
    variables: { interestId, interestType },
  });

  // TODO: deal with loading and error
  if (loading) {
    console.info('Loading');
  }

  if (error) {
    console.error(error, `Error fetching interest data for interest ${interestType} on id ${interestId}`);
  }

  const queryData = get(data, 'interestData', []);

  return children(queryData);
};

export default InterestDataQuery;
