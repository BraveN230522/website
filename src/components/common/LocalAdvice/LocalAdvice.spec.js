import React from 'react';
import { render } from '@testing-library/react';
import withGlobals from '@swiamTestUtils/withGlobals';
import LocalAdvice from './index';
import { mockLocal, mockAdvice } from './mocks';

describe.skip('<LocalAdvice />', () => {
  it('Renders a local advice component', () => {
    const { container } = render(
      <LocalAdvice
        tips={mockLocal.tips}
        local={mockLocal.local}
        adviceImage={mockLocal.adviceImage}
        cardData={mockAdvice}
      />,
      {
        wrapper: withGlobals,
      }
    );
    expect(container).toMatchSnapshot();
  });
});
