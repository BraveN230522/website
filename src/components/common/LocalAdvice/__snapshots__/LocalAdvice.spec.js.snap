// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LocalAdvice /> Renders a local advice component 1`] = `
.emotion-69 {
  background-color: #ffffff;
  max-width: 1255px;
  width: 100%;
  margin-left: auto;
  margin-right: auto;
  padding-left: 30px;
  padding-right: 30px;
  margin-bottom: 64px;
}

.emotion-69 article {
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

@media (max-width:48em) {
  .emotion-69 {
    padding: 0;
  }

  .emotion-69 > header {
    padding: 0 30px;
  }

  .emotion-69 article {
    -webkit-align-items: flex-start;
    -webkit-box-align: flex-start;
    -ms-flex-align: flex-start;
    align-items: flex-start;
  }
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin: 0 0 16px;
}

.emotion-0 {
  color: #000000;
  font-family: Inter,sans-serif;
  font-size: 22px;
  font-weight: 400;
  -webkit-letter-spacing: -0.15px;
  -moz-letter-spacing: -0.15px;
  -ms-letter-spacing: -0.15px;
  letter-spacing: -0.15px;
  line-height: 29px;
  margin: 0;
}

@media (max-width:23.625em) {
  .emotion-0 {
    font-size: 20px;
  }
}

.emotion-2 {
  cursor: pointer;
  font-family: Inter,sans-serif;
  font-size: 14px;
  font-weight: 400;
  line-height: 19px;
  -webkit-letter-spacing: -0.1px;
  -moz-letter-spacing: -0.1px;
  -ms-letter-spacing: -0.1px;
  letter-spacing: -0.1px;
  opacity: 0.5;
  -webkit-transition: all ease-in 0.2s;
  transition: all ease-in 0.2s;
}

.emotion-2:hover {
  opacity: 1;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-66 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-32 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
}

@media (max-width:60em) {
  .emotion-32 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

.emotion-8 {
  background-image: url(https://i.insider.com/5d0150e06fc9200b5a05f042?width=1136&format=jpeg);
  background-repeat: no-repeat;
  background-position: center;
  background-size: cover;
  border-radius: 8px;
  min-height: 357px;
  max-width: 721px;
  margin: 0 27px 0 0;
  width: 100%;
  cursor: pointer;
}

.emotion-8:hover g:nth-of-type(1) > use {
  fill: #ffffff;
}

.emotion-8:hover g:nth-of-type(2) > g use {
  fill: #2891ff;
}

@media (max-width:60em) {
  .emotion-8 {
    margin: 0 0 16px;
    max-width: 100%;
  }

  .emotion-8 svg {
    -webkit-transform: translate(-50%,180%);
    -ms-transform: translate(-50%,180%);
    transform: translate(-50%,180%);
  }
}

@media (max-width:48em) {
  .emotion-8 {
    min-height: 230px;
    margin-left: 30px;
    width: calc(100% - 60px);
  }

  .emotion-8 svg {
    -webkit-transform: translate(-50%,110%);
    -ms-transform: translate(-50%,110%);
    transform: translate(-50%,110%);
  }
}

.emotion-6 {
  cursor: pointer;
  left: 50%;
  position: relative;
  top: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
}

.emotion-6 g,
.emotion-6 use {
  -webkit-transition: all 0.3s ease-in;
  transition: all 0.3s ease-in;
}

.emotion-30 {
  -webkit-align-self: stretch;
  -ms-flex-item-align: stretch;
  align-self: stretch;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  max-width: 447px;
  width: 100%;
}

.emotion-30 > div {
  min-height: 167px;
}

.emotion-30 > div img {
  max-height: 168px;
}

@media (max-width:60em) {
  .emotion-30 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
    -webkit-box-pack: start;
    -webkit-justify-content: flex-start;
    -ms-flex-pack: start;
    justify-content: flex-start;
    max-width: 100%;
    overflow-x: scroll;
  }

  .emotion-30 > div {
    min-height: initial;
    margin-right: 12px;
    width: 50%;
  }

  .emotion-30 > div img {
    max-width: 138px;
  }

  .emotion-30 > div:last-child {
    margin-right: 0;
  }
}

@media (max-width:48em) {
  .emotion-30 > div {
    margin-right: 12px;
    width: 348px;
  }
}

.emotion-18 {
  background-clip: border-box;
  background-color: #f4f4f4;
  border-radius: 8px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  position: relative;
  width: 447px;
}

.emotion-10 {
  border-top-left-radius: 8px;
  border-bottom-left-radius: 8px;
  object-fit: cover;
  max-width: 200px;
}

.emotion-16 {
  padding: 15px 20px;
}

@media (max-width:48em) {
  .emotion-16 {
    padding: 10px 15px;
    width: 227px;
  }
}

.emotion-12 {
  color: #000000;
  font-family: Inter,sans-serif;
  font-size: 17px;
  font-weight: 700;
  line-height: 23px;
  margin-top: 0;
  margin-bottom: 12px;
  -webkit-letter-spacing: -0.1px;
  -moz-letter-spacing: -0.1px;
  -ms-letter-spacing: -0.1px;
  letter-spacing: -0.1px;
  text-overflow: ellipsis;
  overflow: hidden;
  display: -webkit-box;
  -webkit-line-clamp: 2;
  -webkit-box-orient: vertical;
}

.emotion-14 {
  color: #000000;
  font-family: Helvetica,sans-serif;
  font-size: 12px;
  font-weight: 400;
  line-height: 16px;
  -webkit-letter-spacing: -0.08px;
  -moz-letter-spacing: -0.08px;
  -ms-letter-spacing: -0.08px;
  letter-spacing: -0.08px;
  margin-bottom: 0;
}

.emotion-64 {
  background-color: #f4f4f4;
  border-radius: 8px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  min-height: 102px;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin-top: 24px;
  padding: 23px 17px;
  width: 100%;
}

@media (max-width:60em) {
  .emotion-64 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media (max-width:48em) {
  .emotion-64 {
    margin-left: 30px;
    width: calc(100% - 60px);
  }
}

.emotion-42 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 3px;
}

.emotion-42:last-child {
  margin-right: 83px;
}

@media (max-width:74.625em) {
  .emotion-42:last-child {
    margin-right: 32px;
  }
}

@media (max-width:68.125em) {
  .emotion-42:last-child {
    margin-right: 0;
  }
}

@media (max-width:60em) {
  .emotion-42 {
    -webkit-box-pack: start;
    -webkit-justify-content: flex-start;
    -ms-flex-pack: start;
    justify-content: flex-start;
    margin-bottom: 32px;
  }

  .emotion-42:last-of-type {
    margin-bottom: 0;
  }
}

.emotion-34 {
  width: 48px;
  margin-right: 12px;
  cursor: pointer;
}

.emotion-34 use {
  fill: #ffffff;
}

.emotion-34 g,
.emotion-34 use {
  -webkit-transition: all 0.3s ease-in;
  transition: all 0.3s ease-in;
}

.emotion-34:hover use {
  fill: #2891ff;
}

.emotion-34:hover g[fill] {
  fill: #ffffff;
}

.emotion-36 {
  font-family: Inter,sans-serif;
  font-size: 18px;
  font-weight: 500;
  margin-top: 0;
  margin-bottom: 4px;
}

.emotion-38 {
  display: block;
  color: #000000;
  opacity: 0.6;
  font-family: Helvetica,sans-serif;
  font-size: 13px;
  line-height: 13px;
}

<div>
  <div
    class="emotion-68 emotion-69 emotion-70"
  >
    <header
      class="emotion-4 emotion-5"
    >
      <h1
        class="emotion-0 emotion-1"
      >
        Local NYC Advice
      </h1>
      <a
        class="emotion-2 emotion-3"
      >
        More
      </a>
    </header>
    <article
      class="emotion-66 emotion-67"
    >
      <div
        class="emotion-32 emotion-33"
      >
        <div
          class="emotion-8 emotion-9"
        >
          <svg
            class="emotion-6 emotion-7"
            color="#000"
            fill="#000"
            height="72"
            width="72"
          >
            <defs>
              <clippath
                id="ca"
              >
                <use
                  xlink:href="#aa"
                />
              </clippath>
              <clippath
                id="da"
              >
                <use
                  xlink:href="#ba"
                />
              </clippath>
              <path
                d="M36 72c19.882 0 36-16.118 36-36S55.882 0 36 0 0 16.118 0 36s16.118 36 36 36z"
                id="aa"
              />
              <path
                d="M18 0l18 29.647H0L18 0z"
                id="ba"
                transform="rotate(90 19.059 37.059)"
              />
            </defs>
            <use
              fill-opacity=".2"
              xlink:href="#aa"
            />
            <g
              clip-path="url(#ca)"
            >
              <use
                fill="#2891FF"
                stroke="#2891FF"
                stroke-width="4"
                xlink:href="#aa"
              />
            </g>
            <g>
              <use
                fill="#FFF"
                xlink:href="#ba"
              />
              <g
                clip-path="url(#da)"
              >
                <use
                  fill="none"
                  stroke="#FFF"
                  stroke-width="2"
                  xlink:href="#ba"
                />
              </g>
            </g>
          </svg>
        </div>
        <div
          class="emotion-30 emotion-31"
        >
          <div
            class="emotion-18 emotion-19"
          >
            <img
              class="emotion-10 emotion-11"
              src="https://content.r9cdn.net/rimg/kimg/f1/0b/34800236-58e51e7e.jpg?width=1004&crop=true&caller=HotelDetailsPhoto.jpeg"
            />
            <div
              class="emotion-16 emotion-17"
            >
              <h1
                class="emotion-12 emotion-13"
              >
                Restaurants near Madison Square Garden
              </h1>
              <p
                class="emotion-14 emotion-15"
              >
                For years the area around Manhattan’s premier entertainment venue has been criticised for a lack of decent food options.
              </p>
            </div>
          </div>
          <div
            class="emotion-18 emotion-19"
          >
            <img
              class="emotion-10 emotion-11"
              src="https://content.r9cdn.net/rimg/kimg/f1/0b/34800236-58e51e7e.jpg?width=1004&crop=true&caller=HotelDetailsPhoto.jpeg"
            />
            <div
              class="emotion-16 emotion-17"
            >
              <h1
                class="emotion-12 emotion-13"
              >
                Hotels near Madison Square Garden
              </h1>
              <p
                class="emotion-14 emotion-15"
              >
                Now, you might have heard that Manhattan isn’t exactly the cheapest place in the world, and a night or two in a hotel could ..
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="emotion-64 emotion-65"
      >
        <div
          class="emotion-42 emotion-43"
        >
          <svg
            class="emotion-34 emotion-30"
            color="#000"
            fill="#000"
            height="43"
            viewBox="0 0 43 43"
            width="43"
          >
            <defs>
              <clippath
                id="h"
              >
                <use
                  xlink:href="#g"
                />
              </clippath>
              <path
                d="M21.499 0h.002q.528 0 1.055.026.527.026 1.052.078.526.051 1.048.129.522.077 1.04.18.517.103 1.029.231.512.129 1.017.282.505.153 1.002.33.497.179.984.38.488.202.965.428.477.226.943.475.465.248.918.52.452.271.891.564.439.293.863.608.424.314.832.649t.799.69q.39.354.764.727.373.373.728.764.354.391.689.8.335.407.65.831.313.424.607.863.293.439.564.891.272.453.52.918.25.466.475.943.225.477.428.965.202.487.38.984.177.497.33 1.002.154.505.282 1.017.128.512.23 1.03.104.517.181 1.04.078.521.13 1.047.051.525.077 1.052.026.527.026 1.055v.002q0 .528-.026 1.055-.026.527-.077 1.052-.052.526-.13 1.048-.077.522-.18 1.04-.103.517-.231 1.029-.129.512-.282 1.017-.153.505-.33 1.002-.178.497-.38.984-.203.488-.428.965-.226.477-.475.943-.248.465-.52.918-.271.452-.564.891-.294.439-.608.863t-.649.832q-.335.408-.69.799-.354.39-.727.764-.373.373-.764.728-.391.354-.8.689-.407.335-.831.65-.424.313-.863.607-.439.293-.891.564-.453.272-.918.52-.466.25-.943.475-.477.225-.965.428-.487.202-.984.38-.497.177-1.002.33-.505.154-1.017.282-.512.128-1.03.23-.517.104-1.04.181-.521.078-1.047.13-.525.051-1.052.077Q22.029 43 21.5 43h-.002q-.528 0-1.055-.026-.527-.026-1.052-.077-.526-.052-1.048-.13-.522-.077-1.04-.18-.517-.103-1.029-.231-.512-.129-1.017-.282-.505-.153-1.002-.33-.497-.178-.984-.38-.488-.203-.965-.428-.477-.226-.943-.475-.465-.248-.918-.52-.452-.271-.891-.564-.439-.294-.863-.608t-.832-.649q-.408-.335-.799-.69-.39-.354-.764-.727-.373-.373-.728-.764-.354-.391-.689-.8-.335-.407-.65-.831-.314-.424-.607-.863t-.564-.891q-.272-.453-.52-.918-.25-.466-.475-.943-.226-.477-.427-.965-.202-.487-.38-.984t-.331-1.002q-.153-.505-.282-1.017-.128-.512-.23-1.03-.104-.517-.181-1.04-.078-.521-.13-1.047-.051-.525-.077-1.052Q0 22.029 0 21.5v-.002q0-.528.026-1.055.026-.527.078-1.052.051-.526.129-1.048.077-.522.18-1.04.103-.517.231-1.029.129-.512.282-1.017.153-.505.33-1.002.179-.497.38-.984.202-.488.428-.965.226-.477.475-.943.248-.465.52-.918.271-.452.564-.891.293-.439.608-.863.314-.424.649-.832t.69-.799q.354-.39.727-.764.373-.373.764-.728.391-.354.8-.689.407-.335.831-.65.424-.314.863-.607t.891-.564q.453-.272.918-.52.466-.25.943-.475.477-.226.965-.427.487-.202.984-.38t1.002-.331q.505-.153 1.017-.282.512-.128 1.03-.23.517-.104 1.04-.181.521-.078 1.047-.13.525-.051 1.052-.077Q20.971 0 21.5 0z"
                id="g"
              />
            </defs>
            <use
              fill="#FFF"
              xlink:href="#g"
            />
            <g
              clip-path="url(#h)"
            >
              <use
                fill="none"
                stroke="#2891FF"
                stroke-width="4"
                xlink:href="#g"
              />
            </g>
            <g
              fill="#2891FF"
            >
              <path
                d="M18.686 25.564v7.939h-2.461V12.497h5.771q3.37 0 5.223 1.64 1.853 1.641 1.853 4.63 0 2.988-2.059 4.892-2.058 1.904-5.559 1.904h-2.768zm0-10.84v8.613h2.578q2.549 0 3.889-1.165 1.34-1.164 1.34-3.288 0-4.16-4.921-4.16h-2.886z"
              />
            </g>
          </svg>
          <div
            class="emotion-40 emotion-41"
          >
            <p
              class="emotion-36 emotion-37"
            >
              Arthur Ashe Stadium parking?
            </p>
            <span
              class="emotion-38 emotion-31"
            >
              222
               Views • 
              2
               Replies
            </span>
          </div>
        </div>
        <div
          class="emotion-42 emotion-43"
        >
          <svg
            class="emotion-34 emotion-30"
            color="#000"
            fill="#000"
            height="43"
            viewBox="0 0 43 43"
            width="43"
          >
            <defs>
              <clippath
                id="h"
              >
                <use
                  xlink:href="#g"
                />
              </clippath>
              <path
                d="M21.499 0h.002q.528 0 1.055.026.527.026 1.052.078.526.051 1.048.129.522.077 1.04.18.517.103 1.029.231.512.129 1.017.282.505.153 1.002.33.497.179.984.38.488.202.965.428.477.226.943.475.465.248.918.52.452.271.891.564.439.293.863.608.424.314.832.649t.799.69q.39.354.764.727.373.373.728.764.354.391.689.8.335.407.65.831.313.424.607.863.293.439.564.891.272.453.52.918.25.466.475.943.225.477.428.965.202.487.38.984.177.497.33 1.002.154.505.282 1.017.128.512.23 1.03.104.517.181 1.04.078.521.13 1.047.051.525.077 1.052.026.527.026 1.055v.002q0 .528-.026 1.055-.026.527-.077 1.052-.052.526-.13 1.048-.077.522-.18 1.04-.103.517-.231 1.029-.129.512-.282 1.017-.153.505-.33 1.002-.178.497-.38.984-.203.488-.428.965-.226.477-.475.943-.248.465-.52.918-.271.452-.564.891-.294.439-.608.863t-.649.832q-.335.408-.69.799-.354.39-.727.764-.373.373-.764.728-.391.354-.8.689-.407.335-.831.65-.424.313-.863.607-.439.293-.891.564-.453.272-.918.52-.466.25-.943.475-.477.225-.965.428-.487.202-.984.38-.497.177-1.002.33-.505.154-1.017.282-.512.128-1.03.23-.517.104-1.04.181-.521.078-1.047.13-.525.051-1.052.077Q22.029 43 21.5 43h-.002q-.528 0-1.055-.026-.527-.026-1.052-.077-.526-.052-1.048-.13-.522-.077-1.04-.18-.517-.103-1.029-.231-.512-.129-1.017-.282-.505-.153-1.002-.33-.497-.178-.984-.38-.488-.203-.965-.428-.477-.226-.943-.475-.465-.248-.918-.52-.452-.271-.891-.564-.439-.294-.863-.608t-.832-.649q-.408-.335-.799-.69-.39-.354-.764-.727-.373-.373-.728-.764-.354-.391-.689-.8-.335-.407-.65-.831-.314-.424-.607-.863t-.564-.891q-.272-.453-.52-.918-.25-.466-.475-.943-.226-.477-.427-.965-.202-.487-.38-.984t-.331-1.002q-.153-.505-.282-1.017-.128-.512-.23-1.03-.104-.517-.181-1.04-.078-.521-.13-1.047-.051-.525-.077-1.052Q0 22.029 0 21.5v-.002q0-.528.026-1.055.026-.527.078-1.052.051-.526.129-1.048.077-.522.18-1.04.103-.517.231-1.029.129-.512.282-1.017.153-.505.33-1.002.179-.497.38-.984.202-.488.428-.965.226-.477.475-.943.248-.465.52-.918.271-.452.564-.891.293-.439.608-.863.314-.424.649-.832t.69-.799q.354-.39.727-.764.373-.373.764-.728.391-.354.8-.689.407-.335.831-.65.424-.314.863-.607t.891-.564q.453-.272.918-.52.466-.25.943-.475.477-.226.965-.427.487-.202.984-.38t1.002-.331q.505-.153 1.017-.282.512-.128 1.03-.23.517-.104 1.04-.181.521-.078 1.047-.13.525-.051 1.052-.077Q20.971 0 21.5 0z"
                id="g"
              />
            </defs>
            <use
              fill="#FFF"
              xlink:href="#g"
            />
            <g
              clip-path="url(#h)"
            >
              <use
                fill="none"
                stroke="#2891FF"
                stroke-width="4"
                xlink:href="#g"
              />
            </g>
            <g
              fill="#2891FF"
            >
              <path
                d="M18.686 25.564v7.939h-2.461V12.497h5.771q3.37 0 5.223 1.64 1.853 1.641 1.853 4.63 0 2.988-2.059 4.892-2.058 1.904-5.559 1.904h-2.768zm0-10.84v8.613h2.578q2.549 0 3.889-1.165 1.34-1.164 1.34-3.288 0-4.16-4.921-4.16h-2.886z"
              />
            </g>
          </svg>
          <div
            class="emotion-40 emotion-41"
          >
            <p
              class="emotion-36 emotion-37"
            >
              Best bars near Metlife Stadium?
            </p>
            <span
              class="emotion-38 emotion-31"
            >
              123
               Views • 
              13
               Replies
            </span>
          </div>
        </div>
        <div
          class="emotion-42 emotion-43"
        >
          <svg
            class="emotion-34 emotion-30"
            color="#000"
            fill="#000"
            height="43"
            viewBox="0 0 43 43"
            width="43"
          >
            <defs>
              <clippath
                id="h"
              >
                <use
                  xlink:href="#g"
                />
              </clippath>
              <path
                d="M21.499 0h.002q.528 0 1.055.026.527.026 1.052.078.526.051 1.048.129.522.077 1.04.18.517.103 1.029.231.512.129 1.017.282.505.153 1.002.33.497.179.984.38.488.202.965.428.477.226.943.475.465.248.918.52.452.271.891.564.439.293.863.608.424.314.832.649t.799.69q.39.354.764.727.373.373.728.764.354.391.689.8.335.407.65.831.313.424.607.863.293.439.564.891.272.453.52.918.25.466.475.943.225.477.428.965.202.487.38.984.177.497.33 1.002.154.505.282 1.017.128.512.23 1.03.104.517.181 1.04.078.521.13 1.047.051.525.077 1.052.026.527.026 1.055v.002q0 .528-.026 1.055-.026.527-.077 1.052-.052.526-.13 1.048-.077.522-.18 1.04-.103.517-.231 1.029-.129.512-.282 1.017-.153.505-.33 1.002-.178.497-.38.984-.203.488-.428.965-.226.477-.475.943-.248.465-.52.918-.271.452-.564.891-.294.439-.608.863t-.649.832q-.335.408-.69.799-.354.39-.727.764-.373.373-.764.728-.391.354-.8.689-.407.335-.831.65-.424.313-.863.607-.439.293-.891.564-.453.272-.918.52-.466.25-.943.475-.477.225-.965.428-.487.202-.984.38-.497.177-1.002.33-.505.154-1.017.282-.512.128-1.03.23-.517.104-1.04.181-.521.078-1.047.13-.525.051-1.052.077Q22.029 43 21.5 43h-.002q-.528 0-1.055-.026-.527-.026-1.052-.077-.526-.052-1.048-.13-.522-.077-1.04-.18-.517-.103-1.029-.231-.512-.129-1.017-.282-.505-.153-1.002-.33-.497-.178-.984-.38-.488-.203-.965-.428-.477-.226-.943-.475-.465-.248-.918-.52-.452-.271-.891-.564-.439-.294-.863-.608t-.832-.649q-.408-.335-.799-.69-.39-.354-.764-.727-.373-.373-.728-.764-.354-.391-.689-.8-.335-.407-.65-.831-.314-.424-.607-.863t-.564-.891q-.272-.453-.52-.918-.25-.466-.475-.943-.226-.477-.427-.965-.202-.487-.38-.984t-.331-1.002q-.153-.505-.282-1.017-.128-.512-.23-1.03-.104-.517-.181-1.04-.078-.521-.13-1.047-.051-.525-.077-1.052Q0 22.029 0 21.5v-.002q0-.528.026-1.055.026-.527.078-1.052.051-.526.129-1.048.077-.522.18-1.04.103-.517.231-1.029.129-.512.282-1.017.153-.505.33-1.002.179-.497.38-.984.202-.488.428-.965.226-.477.475-.943.248-.465.52-.918.271-.452.564-.891.293-.439.608-.863.314-.424.649-.832t.69-.799q.354-.39.727-.764.373-.373.764-.728.391-.354.8-.689.407-.335.831-.65.424-.314.863-.607t.891-.564q.453-.272.918-.52.466-.25.943-.475.477-.226.965-.427.487-.202.984-.38t1.002-.331q.505-.153 1.017-.282.512-.128 1.03-.23.517-.104 1.04-.181.521-.078 1.047-.13.525-.051 1.052-.077Q20.971 0 21.5 0z"
                id="g"
              />
            </defs>
            <use
              fill="#FFF"
              xlink:href="#g"
            />
            <g
              clip-path="url(#h)"
            >
              <use
                fill="none"
                stroke="#2891FF"
                stroke-width="4"
                xlink:href="#g"
              />
            </g>
            <g
              fill="#2891FF"
            >
              <path
                d="M18.686 25.564v7.939h-2.461V12.497h5.771q3.37 0 5.223 1.64 1.853 1.641 1.853 4.63 0 2.988-2.059 4.892-2.058 1.904-5.559 1.904h-2.768zm0-10.84v8.613h2.578q2.549 0 3.889-1.165 1.34-1.164 1.34-3.288 0-4.16-4.921-4.16h-2.886z"
              />
            </g>
          </svg>
          <div
            class="emotion-40 emotion-41"
          >
            <p
              class="emotion-36 emotion-37"
            >
              Traffic around Citi Field?
            </p>
            <span
              class="emotion-38 emotion-31"
            >
              3087
               Views • 
              1
               Replies
            </span>
          </div>
        </div>
      </div>
    </article>
  </div>
</div>
`;
